common/djangoapps/course_modes/models.py

```diff
@@ -86,6 +86,11 @@ class Meta:
         """ meta attributes of this model """
         unique_together = ('course_id', 'mode_slug', 'currency')
 
+    def save(self, force_insert=False, force_update=False, using=None):
```

Comment:
the django way of doing this seems to be using [`Model.clean`](https://docs.djangoproject.com/en/1.4/ref/models/instances/#django.db.models.Model.clean) - is overriding `save` preferable for this case?